flixel/math/FlxPoint.hx

```diff
@@ -396,6 +410,16 @@ class FlxPoint implements IFlxPooled
 	{
 		return FlxVector.get(x, y);
 	}
+
+	/**
+	 * Function to get the hypotenuse of the right triangle 
+	 * formed by the components of this `FlxPoint`
+	 * @since 4.3.0
+	 */
+	public inline function hypot():Float
```

Comment:
Yeah, I agree with you. You could just call `toVector().length()` instead. I'll close this.